applications/API/endpoints.py

```
@@ -0,0 +1,505 @@
+    bulk_select_related = ["added_by", "category"]
+    bulk_prefetch_related = ["reports"]
+    filter_backends = [DjangoFilterBackend]
+
+    bulk_cache_timeout = 1800
+    cache_invalidation_patterns = [
+        "Insult:*",
+        "bulk:insult*",
+        "categories:*",
+        "users:*:insults*",
+    ]
+    filter_backends = (DjangoFilterBackend,)  # pyrefly: ignore
```

Ruff (0.12.2)

Class field `filter_backends` is defined multiple times

Remove duplicate field definition for `filter_backends`

(PIE794)